<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskWindow</class>
 <widget class="QMainWindow" name="AddTaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>308</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Task</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="typeComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Priority</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityComboBox">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>110</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionTextEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createTaskButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QComboBox" name="projectsComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="usersComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="topicLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>34</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>22</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
